.circle/install-mongo.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e
+
+sudo apt-get purge mongodb-org*
+sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
+echo "deb http://repo.mongodb.org/apt/ubuntu trusty/mongodb-org/3.2 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
```

Comment:
Should be mongodb-org-3.2.list :D. I didn't fix the typo. Me being lazy and all.